target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8068-mr42.dts

```diff
@@ -87,9 +87,17 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
```

Comment:
This is missing `compatible = "nvmem-cells";` in Line 83. I'm not sure whether it's possible to combine it with `compatible = "atmel,24c64";`